### ISSUE TITLE
Support unit test coverage for TypeScript files

### ISSUE THREAD

**Issue by Pablissimo** [Pablissimo/SonarTsPlugin#3]
Support parsing an LCOV for TypeScript files, similar to the ability of the JavaScript plugin

**Comment by nelo112** [comment]
Hi Pablissimo,
does the coverage work for you? I get a NullPointerException since in TsCoverageSensor.java @line123, since
Measure ncloc = context.getMeasure(resource, CoreMetrics.NCLOC);
returns null. In the SonarQube forum the answer was to not use Sensor but DecoratorContext
http://sonarqube-archive.15.x6.nabble.com/SensorContext-getMeasure-returns-null-td5014664.html
I then commented out the few lines that used the Null reference in the hopes that I would get some results, but it won't show any stats about code coverage. (I created the LCOV file using remapIstanbul with the src maps and the .js LCOV from Istanbul).
I wanted to know what's the latest status on test coverage, so I know where to focus on moving forward

Greetings
Manuel
